src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+function employee(info) {
+  const personInfo = {};
+
+  personInfo.name = info.innerText;
+  personInfo.position = info.dataset.position;
+  personInfo.salary = info.dataset.salary;
+  personInfo.age = info.dataset.age;
+
+  return personInfo;
```

Comment:
можна навіть скоротити
```suggestion
return {
  name: info.innerText,
  position: info.dataset.position,
  salary: info.dataset.salary,
  age: info.dataset.age,
}
```